.github/workflows/compile_protos.yml

```diff
@@ -44,17 +44,7 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
           # note: check out fork because we will need it when we push below
           repository: ${{ github.event.pull_request.head.repo.full_name }}
-      - uses: actions/setup-node@v4
-      - uses: actions/setup-go@v5
```

Comment:
getting rid of the setup-node + setup-go actions has caching implications

if builds are substantially slower, would try adding these back